screens/ReceiptScreen.tsx

```diff
@@ -76,62 +95,78 @@ const ReceiptScreen = (props: StartScreenProps) => {
           <View>
             <Image
               style={{ width: imageSize, height: imageSize, resizeMode: "contain", }}
               source={require("../assets/sheep_1.png")}
             />
-            <Text style={{ alignSelf: "center" }}>{props.observationTotal.whiteGreySheepCount}</Text>
+            <Text style={{ alignSelf: "center", fontSize: fontSize }}>{props.observationsTotal.whiteGreySheepCount}</Text>
           </View>
           <View>
             <Image
               style={{ width: imageSize, height: imageSize, resizeMode: "contain", }}
               source={require("../assets/brown-sheep.png")}
             />
-            <Text style={{ alignSelf: "center" }}>{props.observationTotal.brownSheepCount}</Text>
+            <Text style={{ alignSelf: "center", fontSize: fontSize }}>{props.observationsTotal.brownSheepCount}</Text>
           </View>
           <View>
             <Image
               style={{ width: imageSize, height: imageSize, resizeMode: "contain", }}
               source={require("../assets/black-sheep.png")}
             />
-            <Text style={{ alignSelf: "center" }}>{props.observationTotal.blackSheepCount}</Text>
+            <Text style={{ alignSelf: "center", fontSize: fontSize }}>{props.observationsTotal.blackSheepCount}</Text>
           </View>
         </View>
 
-        <View style={{ flexDirection: "row", justifyContent: "space-around", marginBottom: margin }}>
+        <View style={{ flexDirection: "row", justifyContent: "space-around"}}>
           <View>
             <MaterialCommunityIcons name="tie" size={imageSize} color="#05d" />
-            <Text style={{ alignSelf: "center" }}>{props.observationTotal.blueTieCount}</Text>
+            <Text style={{ alignSelf: "center", fontSize: fontSize }}>{props.observationsTotal.blueTieCount}</Text>
           </View>
           <View>
             <MaterialCommunityIcons name="tie" size={imageSize} color="#070" />
-            <Text style={{ alignSelf: "center" }}>{props.observationTotal.greenTieCount}</Text>
+            <Text style={{ alignSelf: "center", fontSize: fontSize }}>{props.observationsTotal.greenTieCount}</Text>
           </View>
           <View>
             <MaterialCommunityIcons name="tie" size={imageSize} color="#f4d528" />
-            <Text style={{ alignSelf: "center" }}>{props.observationTotal.yellowTieCount}</Text>
+            <Text style={{ alignSelf: "center", fontSize: fontSize }}>{props.observationsTotal.yellowTieCount}</Text>
           </View>
           <View>
             <MaterialCommunityIcons name="tie" size={imageSize} color="#d22" />
-            <Text style={{ alignSelf: "center" }}>{props.observationTotal.redTieCount}</Text>
+            <Text style={{ alignSelf: "center", fontSize: fontSize }}>{props.observationsTotal.redTieCount}</Text>
           </View>
           <View style={{ justifyContent: "space-between" }}>
             <AntDesign name="close" size={60} color="black" />
```

Comment:
Resize i henhold til forrige kommentar
```suggestion
            <AntDesign name="close" size={40} color="black" />
```